docs/topics/contributing-starting-preview-server.adoc

```diff
@@ -13,17 +13,28 @@ You can preview the documentation site locally. To do that, perform the followin
 == Procedure
 
 .Running Preview Server Locally
-. As a regular user, run the `$REPO_HOME/scripts/previewDocsServer.sh` script from the repository's base directory:
+. As root, run the `$REPO_HOME/cico_build_deploy.sh` script from the repository's base directory while setting the `CICO_LOCAL` environment variable to `true`:
 +
 --
 [source,options="nowrap"]
 ----
-$ ./scripts/previewDocsServer.sh
+$ sudo CICO_LOCAL=true ./cico_build_deploy.sh
```

Comment:
Does it need to be sudo? I think I use sudo to remove the existing html/ folder only